tvtk/wrapper_gen.py

```diff
@@ -1044,6 +1045,9 @@ def _write_tvtk_method(self, out, vtk_meth, sig=None):
         """
         if not sig:
```

Comment:
I agree that this is easier to read although I do not believe there should be a problem currently as if the user explicitly passed an empty signature string, then clearly that is what they want.  In any case, I think this is safe to change.